packages/shopify-codemod/test/transforms/simplify-?=-if-else-statement.test.js

```diff
@@ -0,0 +1,8 @@
+import 'test-helper';
```

Comment:
💥  I grant you the power to make up names!  💥  

The CoffeeScript docs hint that it's called `conditional existence operator`, but I don't like that.  Just go with `existential-assignment` until Chris suggests something better ;)